python/dgl/mock_sparse2/reduction.py

```diff
@@ -0,0 +1,322 @@
+"""DGL sparse matrix reduce operators."""
```

Comment:
```suggestion
"""DGL sparse matrix reduce operators"""
```
not a valid sentence